tensorflow/tools/docker/parameterized_docker_build.sh

```diff
@@ -299,6 +317,26 @@ else
   sed "s/^RUN git clone --branch=.* --depth=1/RUN git clone --branch=${TF_DOCKER_BUILD_DEVEL_BRANCH} --depth=1/" \
       "${ORIG_DOCKERFILE}" > "${DOCKERFILE}"
 
+  # Modify the devel Dockerfile to specify the MKL bazel build parameters
+  if [[ "${TF_DOCKER_BUILD_TYPE}" == "mkl" ]]; then
+    if sed -i -e 's/bazel build -c opt /bazel build -c opt --config=mkl /g' "${DOCKERFILE}"
```

Comment:
Not a huge fan of using sed on the dockerfile like this. Is there a better way to add in mkl build params?

Reply:
I don't think it's terribly elegant either, but it is what the original authors did for gpu builds in the lines just before this, so I thought it best to follow the model.

Reply:
I was hoping there was a cleaner way to write the dockerfile before merging this. Using "sed" on the very specific line in the docker file feels pretty brittle.

But, Im more than fine merging this before the python impl is ready.